<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DompetKu - Pelacak Pengeluaran</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="formGroupBox">
      <property name="title">
       <string>Detail Pengeluaran</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelNama">
         <property name="text">
          <string>Nama Pengeluaran:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditNama"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labelKategori">
         <property name="text">
          <string>Kategori:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="comboBoxKategori"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelTanggal">
         <property name="text">
          <string>Tanggal:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="dateEditTanggal">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="labelJumlah">
         <property name="text">
          <string>Jumlah (Rp):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="spinBoxJumlah">
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="singleStep">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelKeterangan">
         <property name="text">
          <string>Keterangan:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QTextEdit" name="textEditKeterangan"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="buttonFrame">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnTambah">
         <property name="text">
          <string>Tambah</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPerbarui">
         <property name="text">
          <string>Perbarui</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnHapus">
         <property name="text">
          <string>Hapus</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBersihkan">
         <property name="text">
          <string>Bersihkan</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <set>QAbstractItemView::SelectRows</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionEkspor_ke_CSV"/>
    <addaction name="separator"/>
    <addaction name="actionKeluar"/>
   </widget>
   <widget class="QMenu" name="menuBantuan">
    <property name="title">
     <string>Bantuan</string>
    </property>
    <addaction name="actionTentang"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBantuan"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEkspor_ke_CSV">
   <property name="text">
    <string>Ekspor ke CSV</string>
   </property>
  </action>
  <action name="actionKeluar">
   <property name="text">
    <string>Keluar</string>
   </property>
  </action>
  <action name="actionTentang">
   <property name="text">
    <string>Tentang</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
